tycho-p2/tycho-p2-plugin/pom.xml

```diff
@@ -53,7 +53,11 @@
             <artifactId>tycho-artifactcomparator</artifactId>
             <version>${project.version}</version>
         </dependency>
-
+		<dependency>
+		    <groupId>org.apache.commons</groupId>
+		    <artifactId>commons-lang3</artifactId>
+		    <version>3.12.0</version>
```

Comment:
Should we define this dependency on the root pom as a managed dependency?

Reply:
It's not declared as dep anywhere else so I decided to keep it local for now. IMHO commons-lang3 doesn't bring that much value nowadays so it's better to just rewrite and don't have this dependency most of the time.

Reply:
Of course if we end up depending on it in more places I'll move it managed dep.